Probesys/bileto
# new: Allow to set up alerts on the contracts

## Related issue(s)



https://github.com/Probesys/bileto/issues/442

## How to test manually



- create a contract and set up alerts (e.g. 80% and 100 days)
- make sure to activate at least one alert → check that it appears on the contract page and in the contracts list

## Checklist
